### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by sebdevaux**
- Move helm chart base code from `helm-charts` repository
- Add CI worflow to release an helm chart version
- Update the documentation

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-rpjjqmuoiu.chromatic.com)

### CODE REVIEW

File: release/ci-steps/release-helm.mjs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+await $`git push --set-upstream origin ${gitBranch}`;
+
+const prBody = `
+# New APIM Helm Chart version ${releasingVersion} has been released
+`;
+echo(chalk.blue('# Create PR on Github helm-charts repository'));
+echo(prBody);
+process.env.PR_BODY = prBody;
+
+await $`gh pr create --title "[APIM] Helm charts ${releasingVersion} release" --body "$PR_BODY" --base gh-seb --head ${gitBranch}`;
```
Comment (leleueri) [inline release/ci-steps/release-helm.mjs]:
```suggestion
await $`gh pr create --title "[APIM] Helm charts ${releasingVersion} release" --body "$PR_BODY" --base gh-pages --head ${gitBranch}`;
```
I guess you frogot to change the branch used for your tests here ?

File: apim-result.xml
Comment (leleueri) [inline apim-result.xml:1]: Should we add this file into the gitignore or the place where it is generated is not the right one ? :thinking:

File: release/ci-steps/release-helm.mjs (hunk 1 above)
Comment (sebdevaux) [reply release/ci-steps/release-helm.mjs]: oops, yes! thanks

File: apim-result.xml
Comment (sebdevaux) [reply apim-result.xml:1]: Unused file -> deleted

**Comment by gaetanmaisse** [comment]
I think we can exclude the helm folder from the license check, just add: 
```xml
                        <exclude>helm/**</exclude>
```
to 
https://github.com/gravitee-io/gravitee-api-management/blob/e8a62a21d15f5538aaf117c37615868e527fcf25/pom.xml#L1188

File: apim-result.xml [hunk 2]
```diff
@@ -0,0 +1,661 @@
+<testsuites>
```
Comment (gaetanmaisse) [inline apim-result.xml]: To remove?

Comment (sebdevaux) [reply apim-result.xml]: Done

**Review by leleueri — APPROVED** [review]

**Pull request merged**